daffodil-core/src/main/scala/org/apache/daffodil/runtime1/processors/PackedBinaryTraits.scala

```diff
@@ -111,6 +113,10 @@ abstract class PackedBinaryDecimalBaseParser(
 
     try {
       val dec = toPrimType(context, dis.getByteArray(nBits, start))
+      if (dec.getBigDecimal.signum() == -1 && decimalSigned == YesNo.No) {
+        PE(start, "Packed binary decimal value is negative, but decimalSigned is no.")
```

Comment:
Should we make the error message say `dfdl:decimalSigned` to make it more clear the issue is with a DFDL property.

Also, is there any value in include the actual number in the diagnostics? That might make debugging easier to show what a data parsed to. I'm not sure what our convention is for including parsed values in diagnostics. If we do that in other places, it might be worth adding it here so we're consistent. Same with the other new errors.